### ISSUE TITLE
[Feature Request] Boolean

### ISSUE THREAD

**Issue by grmon** [excalidraw/excalidraw#7909]
First and foremost I <3 excalidraw and it's name. FOSS for the win!

The single thing I am missing is the option to combine forms, if it is not all the boolean operators it would be really nice to be able to combine forms in a new one and have the option to change the combined outline or something.

Love!

**Comment by Mrazator** [comment]
Thanks for the great feedback!

Could you please elaborate a bit about  "option to combine forms"? It's completely clear what you mean by that.

**Comment by grmon** [comment]
Dear Marcel,Please refer to:https://help.figma.com/hc/en-us/articles/360039957534-Boolean-operationsKind regardsGerrit Sent from out‘n‘about. On May 7, 2024, at 23:42, Marcel Mraz ***@***.***> wrote:﻿
Thanks for the great feedback!
Could you please elaborate a bit about  "option to combine forms"? It's completely clear what you mean by that.

**Comment by Mrazator** [comment]
I see what you mean now. Unsure if we would like to pollute the editor with these operations, though it's an interesting suggestion.

Another possible option is implementing a "fill tool", which should allow you to perform such operations (and many more) on your own.

I.e. in the case of an intersection, it would mean filling a transparent colour inside the intersection.

**Comment by Mrazator** [comment]
Yeah, or the old-school MS paint fill tool. It's sort of weird with transparent color as it's not a traditional color, but one we could say here filling the intersection with white color, considering the background is white as well, the result would be the same.
